modules/claims_api/app/sidekiq/claims_api/poa_updater.rb

```diff
@@ -4,7 +4,7 @@
 
 module ClaimsApi
   class PoaUpdater < ClaimsApi::ServiceBase
-    def perform(power_of_attorney_id) # rubocop:disable Metrics/MethodLength
+    def perform(power_of_attorney_id, icn_for_vanotify = nil, rep = nil) # rubocop:disable Metrics/MethodLength
```

Comment:
can't pass this in. gets sent as plaintext in the Sidekiq UI so we'll be showing ICN's for every request.

Reply:
@FonzMP Do you think adding it to the headers here as a value that we can then grab in the job works? [0f8b972](https://github.com/department-of-veterans-affairs/vets-api/pull/18687/commits/0f8b972cc3795cec089a26952cda3dbf1ff09911)